css/style.css

```
@@ -6,14 +6,165 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+header img{
+    width:50px;
+    height:50px;
+}
+header{
+    display:flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+/*nav*/
+.topnav a:hover, .topnav a:focus{
+    color:rgb(255, 123, 0);
+}
+.topnav {
+    overflow: hidden;
+}
+.topnav a {
+    float: left;
+    display: block;
+    color: grey;
+    text-align: center;
+    padding: 14px 16px;
```

```suggestion
    padding: 1rem;
```
We don't have to be pixel perfect, and we should use relative units, so we can instead use `1rem` here